assembl/static2/js/app/components/administration/navbar.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { browserHistory } from 'react-router';
+import { Translate } from 'react-redux-i18n';
+import { Col } from 'react-bootstrap';
+import { getDiscussionSlug, calculatePercentage } from '../../utils/globalFunctions';
+import { get } from '../../utils/routeMap';
+
+class Navbar extends React.Component {
+  constructor(props) {
+    super(props);
+    const { currentStep, totalSteps } = this.props;
+    this.state = {
+      currentStep: currentStep,
+      totalSteps: totalSteps
+    };
+    this.goToSection = this.goToSection.bind(this);
+  }
+  componentWillReceiveProps(nextProps) {
+    this.setState({
+      currentStep: nextProps.currentStep,
+      totalSteps: nextProps.totalSteps
+    });
+  }
+  goToSection(stepNb) {
+    const slug = { slug: getDiscussionSlug() };
+    const { phaseIdentifier } = this.props;
+    browserHistory.push(`${get('administration', slug)}${get('adminPhase', { ...slug, phase: phaseIdentifier })}?section=${stepNb}`);
```

Comment:
This is more of a question than a code-change request. Based on docs, it seems like this is the appropriate way to change pages in `react-router`, wanted to make sure that is what you guys are doing.